forge-gui/res/cardsfolder/upcoming/carnivorous_canopy.txt

```diff
@@ -0,0 +1,7 @@
+Name:Carnivorous Canopy
+ManaCost:2 G
+Types:Sorcery
+A:SP$ Destroy | ValidTgts$ Artifact,Enchantment,Creature.withFlying | TgtPrompt$ Select target artifact, enchantment, or creature with flying. | SubAbility$ DBProliferate | SpellDescription$ Destroy target artifact, enchantment, or creature with flying. 
+SVar:DBProliferate:DB$ Proliferate | ConditionDefined$ Targeted | ConditionPresent$ Permanent.cmcLE3 | SpellDescription$ If that permanent's mana value was 3 or less, proliferate. (Choose any number of permanents and/or players, then give each another counter of each kind already there.)
```

Comment:
@tool4ever in case of clone, what is the cmc of Targeted there? still what it was on the battlefield?

Reply:
yea seems to work